guides/common/modules/ref_pxe-loaders.adoc

```diff
@@ -0,0 +1,84 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="pxe-loaders"]
+= PXE loaders
+
+In {Project}, a PXE loader defines the `filename` DHCPv4 option to use during provisioning.
+{Project} deploys the required files to TFTP {SmartProxy}.
+
+.Limitations
+* {Project} can deploy the required booting files only for the `x86_64` architecture.
+ifdef::satellite[]
+* {Team} does not provide support for iPXE workflows.
+endif::[]
+
+.PXE loaders and DHCPv4 filenames
+|===
+| PXE loader option | `filename` value | Use for
+
+| *None*
+| `""`
+| ?
```

Comment:
I find a test that mentions it's for non-PXE: https://github.com/theforeman/foreman/blob/ee1afb939ebfea220177b5066c4bb9744721cdfe/test/controllers/shared/pxe_loader_test.rb#L24-L28

I'd guess this is used for image based provisioning, but I see it's also used for some exotic operating systems: Solaris, Juniper JunOS, Cisco NX-OS, Huawei VRP. The last 3 all are network devices (routers, switches) which are special.

https://theforeman.org/manuals/3.15/index.html#4.4.3ProvisioningTemplates writes:
> None - no filename passed (e.g. for HTTP booting via iPXE)